src/components/characterComonents.tsx

```diff
@@ -84,7 +78,6 @@ function CharacterComponents({ incBasket, basketAmount }: param) {
 
   return (
     <div>
-      <Basket basketAmount={basketAmount} />
       <div>{data ? `we got the data` : `searching...`}</div>
       <PageAnateComponent />
       <SearchInput handleSearch={handleSearch}/>
```

Comment:
Could be cool to rename this
```suggestion
      <div>{data ? `we got the data` : `searching...`}</div>
      <Paginator/>
      <SearchInput handleSearch={handleSearch}/>
```